Community Choice 2018 (#1967)
ppy/osu-wiki: news/2019-01-16-community-choice-2018.md

---
layout: post
title: "Community Choice: 2018"
date: 2019-01-16 04:00:00 +0000
---

> It is time to take a look at 2018 in retrospective, and decide which 10 maps from each game mode will become the highly vaunted Community Favourites!

Reviewer: ```suggestion
It's time to take a look at 2018 in retrospect, and decide which 10 maps from each game mode will become the highly vaunted Community Favourites!
```
first contraction just to fit the register of the rest of the post

[![](/wiki/shared/news/2019-01-16-community-choice-2018/header.jpg)](https://osu.ppy.sh/community/contests/74)

> If you're an older player, you might be more familiar with the Community Choice voting as the old "best of" contests held long ago around this time of year. Never fear, Community Choice is basically exactly the same thing, except with more style. We decided to change the name from "best of" to encourage a more rounded look at the voting and to have the result better reflect the kind of achievement it truly is.

Reviewer 2: This seems kinda bizarre considering we've already called it community choice once.

Here's how all this goes down:

Every player gets **10 votes per game mode** to cast for their most adored **ranked** maps of 2018 **that they have played at least once**. Any maps you're eligible to vote on will *automagically* show up in the voting listing.

> Loved maps are currently not eligible.

Reviewer 2: This seems kind of hanging out randomly in the middle of the article. Maybe start a new list of bullets?

Find your favorite maps in the running for a given game mode, click them once to shoot them a star of intense approval, and you're done! Easy as that.

The highest voted map from each mode will earn the timeless title of Community Favourite of 2018, and its mapper will score **an exclusive new profile badge, plus 6 months of osu!supporter for their hard-earned efforts.**

> The remaining maps in the top 10 for each mode will also have their mappers receive **3 months of osu!supporter** to commemorate their work.

Reviewer 2: Nitpicky, but 

> their _respective_ mappers 

maybe.

Ready to vote? Check out the links below:

* [osu!](https://osu.ppy.sh/community/contests/74)
* [osu!taiko](https://osu.ppy.sh/community/contests/75)
* [osu!catch](https://osu.ppy.sh/community/contests/76)
* [osu!mania](https://osu.ppy.sh/community/contests/77)

> **The voting will run until the end of January (roughly 14 days)**, so make sure to tell all of your friends and get them in on the voting before the polls close!

Reviewer 2: Exact time not necessary?
